sdk/triggers.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { getTriggersListUrl, getSpecificTriggerUrl } = require('./url');
+const { getValidatedBody } = require('./handleError');
+
+async function upsert(apiKey, triggerName, triggerOptions) {
+  const listOptions = {
+    url: getSpecificTriggerUrl(triggerName),
+    headers: {
+      'X-Binaris-Api-Key': apiKey,
+    },
+    json: true,
+    body: triggerOptions,
+  };
+  return getValidatedBody(listOptions);
+}
+
+async function remove(apiKey, triggerName) {
+  const listOptions = {
+    url: getSpecificTriggerUrl(triggerName),
+    headers: {
+      'X-Binaris-Api-Key': apiKey,
+    },
+    json: true,
+  };
+  return getValidatedBody(listOptions, 'delete');
+}
+
+async function list(apiKey) {
+  const listOptions = {
+    url: getTriggersListUrl(),
+    headers: {
+      'X-Binaris-Api-Key': apiKey,
+    },
+    json: true,
+  };
+  return getValidatedBody(listOptions, 'get');
+}
+
+module.exports = {
```

Comment:
This entire file could a single function that takes in like 3 params and generates a new function.

Reply:
refactored the file a bit, you can take a look

Reply:
much better, thanks